src/uwtools/config/formats/yaml.py

```diff
@@ -116,6 +126,8 @@ def dump_dict(cfg: dict, path: Optional[Path] = None) -> None:
         :param path: Path to dump config to.
         """
         yaml.add_representer(TaggedString, TaggedString.represent)
+        yaml.add_representer(f90nml.namelist.Namelist, represent_namelist)
+        yaml.add_representer(OrderedDict, represent_ordereddict)
```

Comment:
I think we have the same 3 `add_representer()` calls now, both here and in `__repr__()`, if I'm not mistaken. Maybe it's time to factor those out into a method that can be called from both call sites, so that these are kept in-sync, assuming that all 3 should always be called in both cases? We'd need a direct unit test of that new method, but maybe there's logic to factor out of existing tests to create that one; then the ones that (hopefully) currently make assertions about `add_representer()` having been called could mock the new method and assert that it was called.

Reply:
I'm still learning this depth of YAML manipulation, but I believe there's a way to set up a group and call that - I was thinking about it again once they started stacking up, I'll look more into it

Reply:
Hm. It looks like `yaml.add_multi_representer()` doesn't do what I'd like, and a few other options just clutter space with tuples instead. I think factoring it out to another method is the answer, barring any new novel approaches